devops/llvm-anybuild.yml

```diff
@@ -0,0 +1,339 @@
+parameters:
```

Comment:
Running a test build [here](https://dev.azure.com/ProjectVeronaCI/Project%20Verona/_build/results?buildId=2453&view=results)

Reply:
And another one [here](https://dev.azure.com/ProjectVeronaCI/Project%20Verona/_build/results?buildId=2454&view=results)